datalad/support/tests/test_annexrepo.py

```diff
@@ -1864,7 +1864,14 @@ def test_AnnexRepo_dirty(path):
         ok_(repo.dirty)
         # commit
         repo.commit("file2.txt annexed")
-    ok_(not repo.dirty)
+
+    try:
+        ok_(not repo.dirty)
+    except AssertionError:
+        if "7.20191024" <= external_versions['cmd:annex'] < "7.20191230":
```

Comment:
I've tested this locally with git-annex versions within and on each side of this range, but if desired I can also push a temporary commit to this PR so that the test is run with neurodebian "-devel" git-annex version.

Reply:
I think it would be fine...  For some reason I thought that our `external_versions` (well distutils' LooseVersion) can't handle comparison properly with string on the left... but I have tried, seems to work, so may be just some old memory

Looks good otherwise.  Thanks!

Reply:
Hmm, would you prefer me to use another approach just to be safe?

Reply:
well -- may be it is just my bad memory somehow... Left comparison seems to work fine
```
$> python -c 'from distutils.version import LooseVersion as LV; print("123.0" < LV("13.2"))'
False
```
so let's be brave (but not bold! ;))